### DISCUSSION TITLE
Install homebrew on Mac M1

### DISCUSSION THREAD

**Discussion by fredboudin** [Homebrew/discussions#4082] [Getting started]
### Output of `brew config`

```shell
I don't know what to write here ?
```

### Output of `brew doctor`

```shell
I don't know what to write here ?
```

### Description of issue

Hi,
I want to install Homebank on my Mac M1 but I have some questions.
The install is more complicated that on linux because it must be done via homebrew.
How much ram is homebrew and where is it installed on OSX (folder).
How to remove it if needed ?
Will homebank soft updates be automatic or will I have to use a homebrew terminal command to check ?
Can I remove homebrew once Homebank is installed ?

**Comment by SMillerDev** [comment]
> How much ram is homebrew.

That depends on what you're doing with Homebrew.

> where is it installed on OSX (folder)

https://docs.brew.sh/Installation

> How to remove it if needed ?

https://docs.brew.sh/Installation#uninstallation

> Will homebank soft updates be automatic or will I have to use a homebrew terminal command to check ?

Since Homebank is build from source you'll need to use Homebrew to manage it's updates

> Can I remove Homebrew once Homebank is installed ?

I guess you can, but you'd never get updates which is probably bad for banking software.

**Reply by fredboudin** [reply]
thanks for your quick answer.
How can is it done : Since Homebank is build from source you'll need to use Homebrew to manage it's updates ?

**Reply by SMillerDev** [reply]
https://docs.brew.sh/Manpage#upgrade-options-outdated_formulaoutdated_cask-

**Reply by fredboudin** [reply]
sorry, so many options I don't understand how to only check for homebank updates ?
On linux, that was automatic with the update manager and the repositeries.

**Reply by SMillerDev** [reply]
Well, if you only have Homebank installed `brew upgrade` will only upgrade Homebank and it's dependencies since there is nothing else.

**Comment by fredboudin** [comment]
hum, it seems more complicated than I expected.On Homebank site, the procedure is 

using command in terminal

1. proceed homebrew package manager:
$ruby -e "$(curl -fsSL https://raw.github.com/mxcl/homebrew/go)"
2. proceed Homebank:
$brew install homebank

they don't tell about CLT for Xcode and The Bourne-again shell for installation (i.e. bash) ?
Do I really must install Xcode ?

**Reply by SMillerDev** [reply]
No, the XCode CLT is not the same as XCode.

> proceed homebrew package manager:

That is a very very old download link. So old it does not work anymore. The new download link is on brew.sh.

> they don't tell about CLT for Xcode and The Bourne-again shell for installation (i.e. bash) ?

Bash is provided by macOS and the installer will download the XCode CLT if it is not installed already.

---

This is steering towards "Feedback on the Homebank install instructions", which would be better handled here: http://homebank.free.fr/en/support.php

**Comment by fredboudin** [comment]
after typing **/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"**
terminal is stuck at **loading Command Line Tools for Xcode** ???
damn that's several Go !
finally, home-brew installation failed **Failed during: git fetch --force origin refs/heads/master:refs/remotes/origin/master**
so I decided to clean the laptop with this line tool 
https://mac.install.guide/commandlinetools/6.html
but when I type **xcode-select -p**, I get "/Library/Developer/CommandLineTools" Instead of **xcode-select: error: unable to get active developer directory...**
If I go on the developper directory, I can't find any **CommandLineTools** folder !
So it is erased but why the command line **xcode-select -p** says it exist ? What is the problem ?
